76-lynx-boreal/technical_specification.md

```diff
@@ -0,0 +1,553 @@
+# Upload Service Intermediary Revision (Lynx Boreal)
+**Epic Type:** Implementation Epic
+
+Epic planning and implementation follow the
+[Epic Planning and Marathon SOP](https://docs.ghga-dev.de/main/sops/sop001_epic_planning.html).
+
+## Scope
+### Outline:
+The goal of this epic is to overhaul the Upload Controller Service (UCS) as part of the
+new [File Upload concept](https://ghga.pages.hzdr.de/internal.ghga.de/feature_archconcept-file-upload/developer/architecture_concepts/ac007_file_upload/).
+While this concept includes references to future services like the Study Repository Service,
+some of the machinery there is still in the planning phase. This epic will therefore
+rely upon some shortcuts, such as extra Data Steward involvement. When the full
+architectural concept is realized, the appropriate adaptations will follow in a future
+epic.
+
+### Included/Required:
+- Implement new Upload Orchestrator Service as described below
+- Revamp existing UCS logic
+- Adapt WPS for "upload" work packages and tokens
+- Adapt CRS to also manage claims for upload boxes
+- Add new schemas to `ghga-event-schemas`
+
+### Not included:
+- Archive test bed integration
+- Subsequent FIS or IFRS updates for the upload path
+- Any front-end work that may be needed
+- GHGA-Connector update
+
+### Upload Controller Service (UCS)
+
+The UCS is responsible for two things:
+1. Cataloging individual files for a given study without knowing
```

Comment:
I would not try to map these concepts to the GHGA domain too much. Especially the assumptions about the relationship to "study" won't hold, so this might be confusing to future readers.

Reply:
I guess the idea was to help understanding by already mentioning the concrete use case and creating a high-level overview that blends the levels, but OTOH you're right, it might be more confusing to readers than helpful. The assumption that the files belong to a study is irrelevant in this context and we might even change that in the future. We should just call it an upload of associated files.